mlrun/errors.py

```diff
@@ -230,6 +230,10 @@ class MLRunTSDBConnectionFailureError(MLRunHTTPStatusError, ValueError):
     error_status_code = HTTPStatus.BAD_REQUEST.value
 
 
+class MLRunMissingProjectError(MLRunBadRequestError):
+    pass
```

Comment:
instead of specifying the message when raising this error, you can define a default message in the exception class itself
```suggestion
class MLRunMissingProjectError(MLRunBadRequestError):
    def __init__(self, message=None):
        default_message = "Project must be provided"
        super().__init__(message or default_message)
```

and call it as MLRunMissingProjectError()